RESTserver/src/main/java/com/softserve/if072/restservice/dao/mybatisdao/UserDAO.java

```diff
@@ -40,16 +40,16 @@
             @Result(property = "name", column = "name"),
             @Result(property = "email", column = "email"),
             @Result(property = "password", column = "password"),
-            @Result(property = "role", column = "role_id", javaType = Role.class, one = @One(select = "com.softserve.if072.restservice.dao.mybatisdao.RoleDAO.getByID")),
+//            @Result(property = "role", column = "role_id", javaType = Role.class, one = @One(select = "com.softserve.if072.restservice.dao.mybatisdao.RoleDAO.getByID")),
             @Result(property = "isEnabled", column = "is_enabled"),
-            @Result(property = "stores", column = "id", javaType = List.class,
-                    many = @Many(select = "com.softserve.if072.restservice.dao.mybatisdao.StorageDAO.getByUserID")),
-            @Result(property = "products", column = "id", javaType = List.class,
-                    many = @Many(select = "com.softserve.if072.restservice.dao.mybatisdao.ProductDAO.getByUserID")),
-            @Result(property = "carts", column = "id", javaType = List.class,
-                    many = @Many(select = "com.softserve.if072.restservice.dao.mybatisdao.CartDAO.getByUserID")),
-            @Result(property = "categories", column = "id", javaType = List.class,
-                    many = @Many(select = "com.softserve.if072.restservice.dao.mybatisdao.CategoryDAO.getByUserID")),
+//            @Result(property = "stores", column = "id", javaType = List.class,
+//                    many = @Many(select = "com.softserve.if072.restservice.dao.mybatisdao.StorageDAO.getByUserID")),
+//            @Result(property = "products", column = "id", javaType = List.class,
+//                    many = @Many(select = "com.softserve.if072.restservice.dao.mybatisdao.ProductDAO.getByUserID")),
+//            @Result(property = "carts", column = "id", javaType = List.class,
+//                    many = @Many(select = "com.softserve.if072.restservice.dao.mybatisdao.CartDAO.getByUserID")),
+//            @Result(property = "categories", column = "id", javaType = List.class,
+//                    many = @Many(select = "com.softserve.if072.restservice.dao.mybatisdao.CategoryDAO.getByUserID")),
```

Comment:
I commented out it, because some methods of other mappers hadn't been ready and there were several exceptions. I'm going to fix it.